src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```
@@ -1343,6 +1378,29 @@ paths:
                 $ref: "#/components/schemas/TransactionStatusResponse"
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
+  "/transaction/subintent-status":
+    post:
+      operationId: TransactionSubintentStatus
+      summary: Get Transaction Subintent Status
```

MINOR: To avoid confusion between Transaction Intent and Subintent, I suggest we don't include "Transaction" in the summary.

```suggestion
      summary: Get Subintent Status
```